docs/markdown/authoritative/howtos.md

```diff
@@ -161,6 +161,8 @@ setting to an existing resolver:
 recursor=[::1]:5300
 ```
 
+**note**: If `resolver` is unset, ALIAS expension is disabled!
```

Comment:
No `resolver` in 4.0.x, right?

Reply:
correct, but these docs are never published. the 4.0.x docs live in a different branch